atkinson/errors/reporters/logger.py

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env python
+""" Python logging based reporter """
+
+from atkinson.errors.reporters import BaseReport
+from atkinson.logging.logger import getLogger
+
+
+class LoggingReporter(BaseReport):
+    """ Class for the logging based reporter """
+
+    def __init__(self, title, logger):
+        """
+        Class constructor
+        :param logger: The logger to use
+        """
+        self.__title = title
+        self.__log = logger
+
+    @classmethod
+    def new(cls, title, description, config):
+        """
+        Create a new report
+        :param title: The title for the report
+        :param description: A description for the report
+        :param config: Configuration dictionary for the report
+        :return: A LoggingReporter instance
+        """
+        log = getLogger()
+        log.error(f"{title}: {description}")
```

Comment:
Why is this an error?

Reply:
The report data is being emitted as python logging error.